proto/iam/rpc/session_svc.proto

```diff
@@ -35,6 +39,9 @@ service SessionSvc {
 	rpc TryStartSession (SessionSvcStartSessionRequest)	
 		returns (SessionSvcStartSessionResponse);
 		
-	rpc RefreshAccessToken (TokenRefreshRequest)	
+	rpc RefreshAccessToken (TokenRefreshRequest)
+		returns (SessionSvcStartSessionResponse);
+
+	rpc Logout(LogoutRequest)
```

Comment:
My thought here is that the application client might have a logout method call, but I'd like the types micro-service call to be more literal and explain the exact action getting performed.

People who don't care about how we do authentication/authorization (e.g. Patrick) can call Logout() and get on with their lives. If someone is digging into the RPC endpoints, let's make it easy for them to find out what's really going on.